services/common/rs_server_common/authentication/authentication.py

```diff
@@ -108,60 +97,38 @@ def auth_validator(station, access_type):
     def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
-            if settings.CLUSTER_MODE:
-                logger.debug("Starting API-KEY validation.")
-                # Read the full cadip station passed in parameter e.g. INS, MPS, ...
-                # no validation needed for landing pages.
-                if station == "cadip" and access_type != "landing_page":
-                    logger.debug("API key validation for cadip search / download")
```

Comment:
This code is too specific to cadip, I've moved it to cadip_search.py::auth_validation